recipe/meta.yaml

```diff
@@ -32,7 +43,6 @@ requirements:
 outputs:
   - name: libcryptominisat
     build:
-      skip: True  # [win and vc<14]
```

Comment:
I wonder if an output's build.skip section is still required because multi-output feedstocks make conda-build a little bit buggy without build.skip, but maybe only for skipping python versions...